user-registration-app/src/app/views/login/login.component.html

```diff
@@ -37,7 +37,7 @@ <h6 class="header-text text-center">LOGIN</h6>
     </div>
     <!-- end:Main Form -->
     <div class="row">
-      <p class="text-center">Alternately, sign up with</p>
+      <p class="text-center">Alternately, sign in with</p>
```

Comment:
A very minor change, maybe make this "Alternatively, login with" So that it matches the "login" text at the top of the email and password fields